test/tape/pool.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
 const test = require('tape');
+const Promise = require('bluebird');
```

Comment:
Is this right?

Reply:
yeah, removed it at first and then realized it makes tests fail. should revert change altogether